### ISSUE TITLE
Общий сбор 17 апреля!

### ISSUE THREAD

**Issue by shabanika** [Airport-DKO/airport#9]
После пар всей командой едем на м.Севастопольская к Стасу. С ночевкой.

**Comment by skalinichev** [comment]
Не, я пас.

**Comment by shabanika** [comment]
Присутствие всех критически важно. Некому и некогда открывать твой, например, код, чтоб посмотреть, которая из компонент налажала.
(Плюс могут выясняться внезапные вещи, вроде того, что ты вызываешь два трапа, а вот мы видели только по одному трапу и одному погрузчику...)

**Comment by skalinichev** [comment]
Да, мы уже обсудили этот вопрос с Кириллом. Присутствовать я буду, но перспектива тащиться на другой конец Москвы, а тем более "ночёвки" меня совсем не вдохновляет.

**Comment by sprodan** [comment]
Это никому не нравится, если ты об этом. Можно снять коворкинг на ночь,
конечно, где-то в центре, чтоб не надо было тащиться и работать там, но
скидываться врядли кто-то хочет, а сдать в субботу хотят все.
